core/nodejsActionBase/src/service.js

```diff
@@ -143,7 +143,7 @@ function NodeActionService(config) {
                     setStatus(Status.ready);
                 }
                 if (typeof result !== 'object') {
-                    return errorMessage(502, 'The action did not return a dictionary.');
+                    return errorMessage(502, 'The action did not return a dictionary or array.');
```

Comment:
It's affected by https://github.com/apache/openwhisk/pull/5290

Reply:
I opened new PR https://github.com/apache/openwhisk/pull/5306 to fix compatibility issue.
I'll revert this change.